jython/DispatcherSystem/Scheduler.py

```diff
@@ -2024,7 +2024,7 @@ def __init__(self, route, graph, station_from, station_to, no_repetitions, train
 
         # note station_to and station_from are strings, while elements of route are locations
 
-        self.logLevel = 0
+        self.logLevel = 1
```

Comment:
Do you intend to keep loglevel 1 in the test release or is this missed from when you tested this?